Gemfile

```diff
@@ -28,11 +28,7 @@ gem "escape_utils"
 gem "faraday-http-cache"
 gem "faraday_middleware"
 gem "fast_blank"
-gem "fast_xor"
-gem "fast_xs"
 gem "ffi"
-gem "fog-aws"
-gem "fog-google"
```

Comment:
👍 I was wary of cutting this, because I wasn't sure what this had used `fog` for in the first place, but it sure doesn't seem needed anymore